1209/F16A/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Cougar TQS adapter
+owner: uri_ba
+license: MIT
```

Comment:
@Arachnid 
Licencing was clarified, and Board file added to project as requested.